### DISCUSSION TITLE
Please add a search bar for repositories

### DISCUSSION THREAD

**Discussion by OAtulA** [community/community#113643] [Mobile]
Hey on GitHub mobile.
I do not see a search bar for GitHub.

![Screenshot_2024-03-22-13-51-11-035_com.github.android.jpg](https://github.com/community/community/assets/98654618/9fbf612d-eef5-43ed-b0e9-f79d8ca091e7)

I want a search bar here.

Please do it.
It's not a big deal for you. 

Regards OAtulA

**Comment by kangarruu** [comment]
Hello, thanks for writing in and for being a part of the GitHub Mobile community! I've forwarded your feedback to the team so we can consider your request for future work. In the meantime, perhaps the Global Code Search on the Home screen could help with finding code snippets and navigating repositories. We hope this improvement will help you easily locate what you’re looking for!
